packages/giselle/src/engine/generations/generate-text.ts

```diff
@@ -344,7 +346,7 @@ function generationModel(
 			return openai.responses(languageModel.id);
 		}
 		case "google": {
-			return google(languageModel.id);
+			return vertex(languageModel.id) as LanguageModel;
```

Comment:
Ideally, I would like to avoid the need for type assertions, but once we fully migrate to AI Gateway, this code will no longer be necessary, so I think this is fine.